core/test/functional/module/module_spec.js

```diff
@@ -11,42 +11,35 @@ describe('Module', function () {
     before(testUtils.teardown);
 
     describe('Setup', function () {
-        it('should resolve with a ghost-server instance', function (done) {
+        it('should resolve with a ghost-server instance', function () {
             ghost().then(function (ghostServer) {
```

Comment:
Missing `return` here (and in following tests). Assertions are no longer being executed :(